.gitlab-ci.yml

```diff
@@ -231,6 +231,8 @@ build:orig:debian:buster:amd64:
   artifacts:
     paths:
       - output/*
+    exclude:
+      - output/opensource/orig/*
```

Comment:
We need to remove the `exclude` in order to verify that the script now manipulates correctly the files (so for example no orig file is left over in `output/opensource/orig/`.